website/docs/r/event_orchestration_service.html.markdown

```diff
@@ -0,0 +1,202 @@
+  * `url` - (Required) The API endpoint where PagerDuty's servers will send the webhook request.
+  * `auto_send` - (Optional) When true, PagerDuty's servers will automatically send this webhook request as soon as the resulting incident is created. When false, your incident responder will be able to manually trigger the Webhook via the PagerDuty website and mobile app.
+  * `headers` - (Optional) Specify custom key/value pairs that'll be sent with the webhook request as request headers.
+    * `key` - (Required) Name to identify the header
+    * `value` - (Required) Value of this header
+  * `parameters` - (Optional) Specify custom key/value pairs that'll be included in the webhook request's JSON payload.
+    * `key` - (Required) Name to identify the parameter
+    * `value` - (Required) Value of this parameter
+* `severity` - (Optional) sets Severity of the resulting alert. Allowed values are: `info`, `error`, `warning`, `critical`
+* `event_action` - (Optional) sets whether the resulting alert status is trigger or resolve. Allowed values are: `trigger`, `resolve`
+* `variables` - (Optional) Populate variables from event payloads and use those variables in other event actions.
+  * `name` - (Required) The name of the variable
+  * `path` - (Required) Path to a field in an event, in dot-notation. This supports both PagerDuty Common Event Format [PD-CEF](https://support.pagerduty.com/docs/pd-cef) and non-CEF fields. Eg: Use `event.summary` for the `summary` CEF field. Use raw_event.fieldname to read from the original event `fieldname` data.
+  * `type` - (Required) Only `regex` is supported
+  * `value` - (Required) The Regex expression to match against. Must use valid [RE2 regular expression](https://github.com/google/re2/wiki/Syntax) syntax.
+* `extractions` - (Optional) Replace any CEF field or Custom Details object field using custom variables.
+  * `template` - (Optional) A value that will be used to populate the `target` field. The configuration can include variables extracted from the payload by using string interpolation. Eg: If you have defined a variable called `hostname` you can set extraction `template` to `High CPU on variables.hostname server` to use the variable in extraction.  This field can be ignored for `regex` based replacements.
+  * `target` - (Required) The PagerDuty Common Event Format [PD-CEF](https://support.pagerduty.com/docs/pd-cef) field that will be set with the value from the `template` or based on `regex` and `source` fields.
+  * `regex` - (Optional) The conditions that need to be met for the extraction to happen. Must use valid [RE2 regular expression](https://github.com/google/re2/wiki/Syntax) syntax. This field can be ignored for `template` based replacements.
+  * `source` - (Optional) Field where the data is being copied from. Must be a PagerDuty Common Event Format [PD-CEF](https://support.pagerduty.com/docs/pd-cef) field. This field can be ignored for `template` based replacements.
```

Comment:
**Blocker:** Again, trying to re-use the API doc wording here. Also seems worth linking to the PCL Path docs. Maybe something like this?

```suggestion
  * `source` - (Optional) The path to the event field where the `regex` will be applied to extract a value. You can use any valid [PCL path](https://developer.pagerduty.com/docs/ZG9jOjM1NTE0MDc0-pcl-overview#paths) like `event.summary` and you can reference previously-defined variables using a path like `variables.hostname`. This field can be ignored for `template` based extractions.
```